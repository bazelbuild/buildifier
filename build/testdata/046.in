foo.bar.baz(
    name = "x",
    c = 0,
    b = 1,
    d = 2,
    a = 3,
)
