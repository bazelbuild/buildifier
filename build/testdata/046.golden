foo.bar.baz(
    name = "x",
    a = 3,
    b = 1,
    c = 0,
    d = 2,
)
